pkg/manifests/manifests_test.go

```diff
@@ -4953,3 +4953,82 @@ xx_omitted_before_deploy__test_file_name:foo.yaml`,
 		})
 	}
 }
+
+func TestSetTLSSecurityConfiguration(t *testing.T) {
+	tests := []struct {
+		name               string
+		apiServerConfig    *APIServerConfig
+		initialArgs        []string
+		tlsCipherSuitesArg string
+		tlsMinVersionArg   string
```

Comment:
(nit) Since these fields remain unchanged throughout the cases here, can we hardcode them in the `require` instead? Or we could change them between `--web.tls-cipher-suites=` (dot-based flags) and `--tls-cipher-suites=` to test both cases that `setTLSSecurityConfiguration` handles.

Reply:
no need I think, `setArg` is already tested in `TestSetArg` e.g.

Reply:
I can hard code them of course.